cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -136,6 +138,28 @@ func (pe *ProbeEvent) deleteBlockDeviceEvent(msg controller.EventMessage) {
 	}
 }
 
+func (pe *ProbeEvent) changeBlockDeviceEvent(msg controller.EventMessage) {
+	var err error
+
+	if msg.AllBlockDevices {
```

Comment:
Can you add additional logs to know about which device is getting processed etc. Ref the `addBlockDeviceEvent()`

Reply:
@z0marlin Please add this log, so that its easier to see the events processed from the logs.

Reply:
done